content/docs/integrations/kubernetes/index.md

```diff
@@ -10,423 +10,177 @@ integrationType: OSS plugin
+2. Navigate to `Outputs` after the Stack has finished, and copy the `RoadieIamRoleArn` and `ExternalID` into the role section of the EKS Kubernetes Settings page in Roadie.
 
-5. Click the checkbox beside "Require External ID" and enter some value.
-
-6. Ignore attached policies and click on ”Next: tags” (Roadie does not need to read your AWS resources, only access to your Kubernetes cluster).
-
-Optional: Add a tag, Key: `3rdPartyIntegration` Value: `Roadie`
-
-7. Click ”Next Review”
-
-8. For the ”Role Name” enter: ”**[your company name]**-roadie-read-only-role”
-
-> Note: ”**[your company name]**” should be replaced by the lowercased value of your company and must follow the convention highlighted above. If it does not follow the convention, the role cannot be assumed. This is for security reasons.
-
-9. For the ”Role description” enter suggested description
-
-```
-This is a role that will be assumed by roadie to gather information on our Kubernetes clusters.
-```
-It should look like this
-
-![role-confirmation](./role-confirmation.png)
-
-10. Click ”Create role”. Your cross federation role is now created.
-
-### Step 2: Modifying trust relationships to only include the new role
-
-1. Get the role of the Roadie backend. This is displayed on the Kubernetes config page `/administration/settings/plugins/kubernetes`.
-
-![AWS roadie role](./role-roadie.png)
-
-2. Search for IAM in the services box and then click on ”Roles” on the left handside tab.
-
-3. Search for your newly created role: ”**[your company name]**-roadie-read-only-role” and click on it.
-You should see a page like below. Record the role arn. This is your assumable role arn which you'll need to configure RBAC in kubernetes.
-
-![role-page](./role-page.png)
-
-3. Click on ”Trust Relationships”, then ”Edit relationship” and add the text below:
-
-``` json
-{
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-      "Effect": "Allow",
-      "Principal": {
-        "AWS": [
-          "ACCOUNT ARN SUPPLIED ON THE CONFIGURATION PAGE"
-        ]
-      },
-      "Action": "sts:AssumeRole",
-      "Condition": {
-        "StringEquals": {
-          "sts:ExternalId": "EXTERNAL ID AS CONFIGURED ABOVE"
-        },
-        "StringLike": {
-          "aws:PrincipalArn": [
-            "*[Roadie backend role name]"
-          ]
-        }
-      }
-    }
-  ]
-}
+### Step 2. Create the Cluster Role and binding.
+1. Connect to your EKS cluster [in your terminal](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html). 
+2. [Apply](https://helm.sh/docs/intro/install/) the Helm Chart to your cluster:
+```shell
+helm repo add roadie https://charts.roadie.io
+helm install roadie-kubernetes-cluster-access roadie/roadie-kubernetes-cluster-access \
+# Only set the following if using the Broker:
```

Comment:
We are in the eks section so these comments should be removed. We can mention at the start of this secion that if they want to connect eks using the broker, then they should go to the broker section.